Form/Type/SearchType.php

```diff
@@ -17,7 +19,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'term',
-                'text',
+                TextType::class,
```

Comment:
This is deprecated and will be removed in 3.0, see https://github.com/symfony/symfony/blob/master/UPGRADE-3.0.md#form